src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,75 @@
+package duke;
+
+import java.util.Scanner;
+
+/**
+ * A class to handle the input and output of user.
+ */
+public class Ui {
+
+    private static final String HORIZONTAL_LINE = "_____________________________________________________________________";
+    private static final String LOGO = " ____        _        \n"
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n";
+
+    private static final String GREETING = "Hello! I am \n"
+            + LOGO
+            + "The awesome bot helper! \n"
+            + "How can I help you today?";
+
+    private static final String GOODBYE = "Bye. Hope to see you again soon!";
+    private final Scanner myScanner;
+
+    /**
+     * Creates a new Ui instance with a new scanner instance
+     **/
+    public Ui() {
+        myScanner = new Scanner(System.in);
+    }
+
+    /**
+     * Add the top an bottom border to the output.
+     *
+     * @param output The message which should be displayed in the output.
+     */
+
+    private static void reply(String output) {
+        System.out.println(HORIZONTAL_LINE);
+        System.out.println(output);
+        System.out.println(HORIZONTAL_LINE);
+    }
+
+    /**
+     * Prints greetings & goodbyes
+     **/
+    public void greet(Boolean isGreeting) {
+        String printMessage = isGreeting
+                ? GREETING
+                : GOODBYE;
+        reply(printMessage);
+    }
+
+    /**
+     * Returns the next line of user input
+     **/
+    public String getInput() {
+        return myScanner.nextLine();
+    }
+
+    /**
+     * Prints a string with proper format
+     **/
+    public void print(String s) {
+        reply(s);
+    }
```

Comment:
Can be renamed better! This method seems to do the same as `reply` so perhaps you could just use one of them?